packages/connector/src/services/core/services/accounts.ts

```diff
@@ -27,17 +37,13 @@ export interface Transaction {
 export interface IlpAccount {
   accountId: string
   parentAccountId?: string
-  disabled: boolean // you can fetch config of disabled account but it will not process packets
+  disabled?: boolean // you can fetch config of disabled account but it will not process packets
```

Comment:
Why is this optional? The migration defines it as `notNullable`.

Reply:
I think I did so `disabled` would be optional with in `createAccount` (with `.defaultTo(false)` in the migration).
I can either just make in optional in `CreateOptions` below or require it to be specified when creating an account.

Reply:
> make it optional in `CreateOptions`

:+1: